### DISCUSSION TITLE
Getting started with PRISM in .Net MAUI

### DISCUSSION THREAD

**Discussion by souravanand001** [PrismLibrary/Prism#2857] [.NET MAUI]
Now that Prism.MAUI is moved to this repo. Is there any documentation or sample application to get started with? For example which nuget should be used for MAUI?

**Comment by giuseppenovielli** [comment]
Hi @souravanand001 ,
check my project sample MAUI7 -> https://github.com/giuseppenovielli/MAUI_XF-Prism-Full-Navigation-Example/tree/master
